packages/webpack/test/e2e-layers.spec.js

```diff
@@ -0,0 +1,87 @@
+const test = /** @type {import('ava').TestFn} */ (require('ava'))
```

Comment:
(suggestion) I'm not certain, but I don't think these tests are type-checked. If adding the directive causes type errors, then they weren't. 😄 I think you might need to type-assert `require('ava')` as `unknown` before type-asserting it to `TestFn`.

also: `@import`

```suggestion
// @ts-check

const test = /** @type {import('ava').TestFn} */ (require('ava'))
```

Reply:
it does change something, but changes to tsconfig would be required anyway - it starts complaining about the ava line first.